content/docs/dvclive/integrations/mmcv.md

```diff
@@ -0,0 +1,64 @@
+# MMCV
+
+The DVCLive - MMCV integration allows you to easily add experiment tracking
+capabilities to your OpenMMlab projects.
+
+The integration is
+[maintained in the MMCV repository](https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/hooks/logger/dvclive.py).
+
+## About MMCV
```

Comment:
Do we need this section? I thought the basic idea was for someone who is already using mmcv to find this integration to hook into dvclive. I don't see sections like this in https://docs.wandb.ai/guides/integrations or https://mlflow.org/docs/latest/tracking.html#automatic-logging.

Reply:
It could probably be removed (at least the extensive list). It makes sense to assume that the user would be already familiar with the library.

Regarding what other ml loggers do:

`mlflow` seems to be the only one without any introduction for the integrations.

`wandb` has small introductions for some of the integrations: https://docs.wandb.ai/guides/integrations/lightning ; https://docs.wandb.ai/guides/integrations/huggingface ; https://docs.wandb.ai/guides/integrations/spacy

`neptune` has small introductions for pretty much all of it's integrations: https://docs.neptune.ai/integrations-and-supported-tools/intro